styles/style.scss

```diff
@@ -0,0 +1,74 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    width: 60%;
+    margin: 0 auto;
+    padding: 3rem;
+}
+
+a {
+    color: black;
+    text-decoration: none;
+}
+
+.header {
+    &__container {
+        padding: 2rem 1rem;
+    }
+
+    &__nav {
+        display: flex;
+        justify-content: space-between;
+        background: whitesmoke;
+        align-items: center;
+    }
+
+    &__logo {
+        display: flex;
+        align-items: center;
+    }
+}
+
+.task-board {
+    &__container {
+        padding: 1rem 1rem;
+        background-color: #f8f9ff;
+    }
+    &__button-area {
+        display: flex;
+        justify-content: space-between;
+        padding-top: 2rem;
+        align-items: center;
+    }
+}
+
+.task {
+    &__container {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+        gap: 1rem 2rem;
+        padding: 1rem 1rem;
+        background-color: #f8f9ff;
+    }
+    &__card {
+        background-color: #ffffff;
+        height: auto;
+        padding: 4rem 1rem;
+        margin-bottom: 1rem;
+        text-align: center;
+        border: 1px solid gray;
```

Comment:
let's take a variable in a separate file

Reply:
done vaiya